distributions/io.jamulus.Jamulus.prepare.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/python3
```

Comment:
Probably you could revert all the changes in this file since we don't use flatpak anymore.

Reply:
Removed this file